processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -666,6 +666,30 @@ func removeUnnecessaryPodData(pod *api_v1.Pod, rules ExtractionRules) *api_v1.Po
 	return &transformedPod
 }
 
+// parseAttributesFromImage parses the image name and tag for differently-formatted image names.
+// returns "latest" as the default if tag not present. also checks if the image contains a digest.
+// if it does, no latest tag is assumed.
```

Comment:
some small nits
// parseNameAndTagFromImage parses the image name and tag for differently-formatted image names.
// returns "latest" as the default if tag not present. It also checks if the image contains a digest,
// if it does, no latest tag is assumed.

Reply:
updated